src/components/ResourceForm/UserFormPage.tsx

```diff
@@ -160,8 +176,13 @@ const UserFormPage = ({
           <FormField
             label="*Email:"
             isValidated
-            validators={['required', 'isEmail']}
-            errorMessages={['This is a mandatory field', 'Invalid email']}
+            disabled={mode === 'edit'}
+            validators={['required', 'isEmail', 'isUniqueEmail']}
+            errorMessages={[
```

Comment:
something we might want to look into for the future might be to refactor these strings into a constants file. Thoughts @petorsong?

Reply:
Let's not pre-emptively handle that - we will have a ticket later on that handles error states/messages across the board, we can consider this more at that point